### PR TITLE
Added she code africa contribution blog article to resolve issue #1930

### LINKED ISSUE #1930: [ Blog ]Add the She Code Africa Contributhon Boot Camp
**Description**
The Layer5 Blog site is lacking recounts of our participation in She Code Africa Contributhon 2020

**Expected Behavior**
Upload a blog post on the [Layer5 site](https://layer5.io/blog), giving recounts of our participation in She Code Africa Contributhon Boot Camp.
You will find the draft on [here](https://docs.google.com/document/d/1IplKOuPX-QQ0htZ8WrGOL2QRT_hQD7n_RXwCzz6J1Hs/edit#heading=h.57y4l6w9m4xz)

**Enviroment:**
- Host OS: 
- Meshery Version: 
- Kubernetes Version: 
- Browser: 

---
**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs).

### PR DESCRIPTION
**Pull request by Jordan-Rob**
Signed-off-by: jordanrob709 <jordanrob709@gmail.com>

**Description**

This PR fixes #1930 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ *] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 790c1d829c9555c806fbb16cb08a80b3377d86c2 at: https://60e5876a88c051488e0e5639--layer5ng.netlify.app

**Comment by Jordan-Rob** [comment]
@Jashpatel1 could you please provide some guidance on how I can accomplish revert changes to package-lock.json specifically. Resources I came across only provide guidance on reverting a whole commit in a remote repo

**Review by Anita-ihuman — COMMENTED** [review]
- You spelt Anita Ihuman as Anitah Ihuman

File: src/collections/blog/2021-06-25-she-code-africa-contribution-boot-camp/index.mdx [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+import Blossom from "./blossomSCA.jpg";
+import Joy from "./joySCA.jpg";
+import Naimat from "./naimatSCA.jpg";
+
+import { BlogWrapper } from "../Blog.style.js";
+
+<BlogWrapper>
+
+  Layer5 took part as one of the organisations that participated in the [She Code Africa Contribution](https://sites.google.com/shecodeafrica.org/Contribution) which started on April 1, 2021, 
+  and it was indeed a pleasant and impactful experience for both the mentors and the mentees from [She Code Africa](https://www.shecodeafrica.org/).
```
Comment (Anita-ihuman) [inline src/collections/blog/2021-06-25-she-code-africa-contribution-boot-camp/index.mdx:31]: Start this as a new sentence by removing and like so.

It was indeed an ...

**Review by Anita-ihuman — COMMENTED** [review]
There is no space between these subheadings  and the content under them.
D49F53A7-2FEF-4730-A1D2-27C4AF7CC386
Space between the headings and the content within them and above them
F8D607D5-C0D7-4864-AB75-33A367864139

Make the mentor(s)bold.
BBF961DC-9A1D-4389-9E09-AB702436ABD4

**Comment by Jordan-Rob** [comment]
@Anita-ihuman let me work on it

**Comment by leecalcote** [comment]
@Jordan-Rob hang in there. There are many people willing to help as and when you might need it. The Websites meeting on Mondays is a great place for real-time troubleshooting (should that ever be helpful).

**Comment by l5io** [comment]
🚀 Preview for commit 0648b4820c04211e8f6cc7a1ab468f6d9d1709ef at: https://60e6145b776ae4f74f7c0391--layer5ng.netlify.app

**Review by Anita-ihuman — APPROVED** [review]
This looks good to me.
@adithyaakrishna  what do you think?

**Comment by Jordan-Rob** [comment]
@Jashpatel1 I made the changes you requested still waiting for your review and feedback so that pull request is merged

**Review by Jashpatel1 — CHANGES_REQUESTED** [review]
There is a common mistype. The program name was contributhon not contribution.

**Comment by leecalcote** [comment]
The finish line is in sight, @Jordan-Rob 😄

**Comment by Jordan-Rob** [comment]
Hi @Jashpatel1, I corrected the typos please review changes made

**Comment by l5io** [comment]
🚀 Preview for commit 3cfada2289227442d72448c7b04f1d86e11453da at: https://60f15dfc203628ed4e17a3fa--layer5ng.netlify.app

**Comment by leecalcote** [comment]
Screen Shot 2021-07-16 at 5 02 48 PM

Accidental formatting issue.

**Comment by leecalcote** [comment]
hyperlinks to the mentees should point to their profiles on layer5.io

**Comment by leecalcote** [comment]
Same for links to mentors and meshmates

**Comment by leecalcote** [comment]
I tossed in a few changes.

File: src/collections/blog/2021-06-25-she-code-africa-contributhon-boot-camp/index.mdx [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+    Layer5 Community Handbook is a curated Document with the ABC's of the community. In this project, mentees were expected to look for repetition pattern across existing docs thereby, creating a structured guide for interaction across all projects.
+  </p>
+  <b>Mentors:</b>
+  <ul>
+    <li><Link to="/community/members/ruth-ikegah">Ruth Ikegah</Link></li>
+    <li><Link  to="/community/members/anita-charles">Anita Ihuman</Link></li>
+  </ul>
+  <b>Mentee(s):</b>
+  <ul>
+    <li><Link to="community/members/anisat-akinbani">Anisat Akinbani</Link></li>
```
Comment (Jashpatel1) [inline src/collections/blog/2021-06-25-she-code-africa-contributhon-boot-camp/index.mdx:52]: This needs to be updated. It points to 404. @Anita-ihuman Can you create an issue to track and solve this?

Comment (leecalcote) [reply src/collections/blog/2021-06-25-she-code-africa-contributhon-boot-camp/index.mdx:52]: Oops. That is a bug that I introduced. It should be: <a href="https://twitter.com/AnisatAkinbani">Anisat Akinbani</a>

**Comment by Anita-ihuman** [comment]
sure.

**Pull request merged**